regression/scripts/Vehicles/BWB.py

```diff
@@ -0,0 +1,423 @@
+# BWB.py
```

Comment:
this file is kind of gross looking. It could use a clean up.

It's also kind of confusing to have the BWB 450 and this BWB together

Reply:
I could remove this and run with the 450 only.

Reply:
NVM